Makefile

```diff
@@ -67,30 +74,15 @@ quick_checks:
 
 flake8:
 	@python tools/actions_local_runner.py \
-		--file .github/workflows/lint.yml \
 		--file-filter '.py' \
 		$(CHANGED_ONLY) \
-		--job 'flake8-py3' \
-		--step 'Run flake8'
-	@python tools/actions_local_runner.py \
-		--file .github/workflows/lint.yml \
-		--file-filter '.py' \
-		$(CHANGED_ONLY) \
-		--job 'flake8-py3' \
-		--step 'Fail if there were any warnings'
```

Comment:
Does this mean the local linter will no longer show Flake8 as "failed" when there are warnings? Or does it just look to see whether there is output or not?

Reply:
For flake8 it doesn't look at `lint.yml` at all, it just calls `$ flake8` directly (or `flake8 <flake8-able files>` for `make quicklint), so it uses flake8's normal error reporting since it's not going through a pipe